packages/react/src/components/Field/FieldCounter.tsx

```diff
@@ -7,21 +7,40 @@ import {
 import { isInputLike } from './fieldObserver';
 
 export type FieldCounterProps = {
-  /** Label template for when `maxCount` is exceeded
+  /**
+   * Label template for when `maxCount` is exceeded.
+   * Use `%d` to insert the number of characters.
```

Comment:
```suggestion
   * Use `%d` for string interpolating the number of characters.
```

Reply:
I was trying to avoid cumbersome words, do you have another suggestion?